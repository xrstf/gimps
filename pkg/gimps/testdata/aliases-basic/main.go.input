package main

import (
	"fmt"
	"k8s.io/api/core/v1"
	"k8s.io/api/networking/v1beta1"
	"log"
)

func main() {
	fmt.Println(v1.Config{})
	fmt.Println(v1beta1.Config{})
	log.Println("foo")
}
