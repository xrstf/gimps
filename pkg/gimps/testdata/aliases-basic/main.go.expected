package main

import (
	"fmt"
	"log"

	corev1 "k8s.io/api/core/v1"
	networkingv1beta1 "k8s.io/api/networking/v1beta1"
)

func main() {
	fmt.Println(corev1.Config{})
	fmt.Println(networkingv1beta1.Config{})
	log.Println("foo")
}
